scripts/pm/hf_from_gcs.py

```diff
@@ -0,0 +1,224 @@
+from google.cloud import storage
+import os
+import re
+import subprocess
+import tempfile
+import shutil
+from collections import defaultdict
+import argparse
+
+# List of GCS directories to check
+dirs = [
+    "gs://marin-eu-west4/checkpoints/llama-8b-tootsie-0.001-19ad63/hf/",
+    "gs://marin-us-central2/checkpoints/llama-8b-tootsie-phase2/hf/",
+    "gs://marin-us-central2/checkpoints/llama-8b-tootsie-phase3/hf/",
+    "gs://marin-us-central2/checkpoints/tootsie-8b-soft-raccoon-3/hf/",
+    "gs://marin-us-central2/checkpoints/llama-8b-tootsie-adept-phoenix/hf/",
+    "gs://marin-us-central2/checkpoints/tootsie-8b-sensible-starling/hf/",
+    "gs://marin-us-central1/checkpoints/tootsie-8b-deeper-starling/hf/",
+]
+
+
+def list_gcs_directories(gcs_path):
```

Comment:
Do we have a utility that does this via fsspec utils?

Reply:
we do not